docs/how-to-use-python-client.md

```diff
@@ -5,39 +5,39 @@ date: 2024-05-09
 keyword: Gravitino Python client
 license: This software is licensed under the Apache License version 2.
 ---
-# Apache Gravitino Python client
+# Apache Gravitino™ (incubating) Python client
 
-Apache Gravitino is a high-performance, geo-distributed, and federated metadata lake.
-It manages the metadata directly in different sources, types, and regions, also provides users
-the unified metadata access for data and AI assets.
+Apache Gravitino (incubating) is a high-performance, geo-distributed, federated metadata lake.
+It manages the metadata directly in different sources, types, and regions, and also provides users
+with unified metadata access for data and AI assets.
 
 Gravitino Python client helps data scientists easily manage metadata using Python language.
 
-![gravitino-python-client-introduction](./assets/gravitino-python-client-introduction.png)
+![gravitino-python-client-introduction](https://github.com/apache/gravitino/blob/main/docs/assets/gravitino-python-client-introduction.png?raw=true)
```

Comment:
Please don't change all the links, we will automatically convert them when publishing with script.